apps/product/tasks.py

```
@@ -1,207 +1,983 @@
+                                company=company,
+                                date=date,
+                                warehouse=warehouse,
+                                marketplace_type="yandexmarket"
+                            )
+
+                    date = date + timedelta(seconds=1)
+    return "succes"    
+
+def get_warehouse_name(business_id,headers, warehouse_id):
+    warehouse_by_busness_id_url = f"https://api.partner.market.yandex.ru/businesses/{business_id}/warehouses"               
+    warehouse_url = f"https://api.partner.market.yandex.ru/warehouses"    
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)